package.json

```diff
@@ -61,7 +61,10 @@
       "json",
       "node"
     ],
-    "moduleNameMapper": { "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/assetsTransformer.js", "\\.(css|less)$": "<rootDir>/assetsTransformer.js" }
```

Comment:
looks like theres in issue in `main` where this linting fix wasn't committed. my bad, will be fixed as soon as either this or #27 is merged